### PR TITLE
[mlir][vector] Add more tests for ConvertVectorToLLVM (7/n)

### PR DESCRIPTION
**Pull request by banach-space**
Adds tests with scalable vectors for the Vector-To-LLVM conversion pass.
Covers the following Ops:
  * vector.fma
  * vector.reduce

### CODE REVIEW

File: mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir [hunk 1]
```diff
@@ -2001,6 +2001,37 @@ func.func @vector_fma(%a: vector<8xf32>, %b: vector<2x4xf32>, %c: vector<1x1x1xf
   return %0, %1, %2, %3: vector<8xf32>, vector<2x4xf32>, vector<1x1x1xf32>, vector<f32>
 }
 
+func.func @vector_fma_scalable(%a: vector<[8]xf32>, %b: vector<2x[4]xf32>, %c: vector<1x1x[1]xf32>, %d: vector<f32>) -> (vector<[8]xf32>, vector<2x[4]xf32>, vector<1x1x[1]xf32>) {
+  // CHECK-LABEL: @vector_fma
```
Comment (nujaa) [inline mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir:2005]:
```suggestion
  // CHECK-LABEL: @vector_fma_scalable
```

**Review by nujaa — COMMENTED** [review]
Hello, LGTM notwithstanding this minor oversight.

File: mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir (hunk 1 above)
Comment (banach-space) [reply mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir]: Great catch, thanks! That's now fixed.

**Review by nujaa — APPROVED** [review]
LGTM.

**Pull request merged**